### PR TITLE
Script edits for growth test (DO NOT MERGE!)

### PR DESCRIPTION
**Pull request by hugobowne**
LETS DO THIS.

### CODE REVIEW

**Comment by hugobowne** [comment]
@westonstearns @adrian-datacamp 

edits were relatively straightforward. of course we won't merge this but i issued PR so we can discuss the edits I made via the diff.

File: scripts/chapter2_script.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ We're dealing with a new Python type here, next to the strings, booleans, intege
 
 ## Subsetting lists
 
-After you've created your very own Python list, you might wonder how you can access information in the list. Python uses the index to do this. Have a look at the fam list again here. The first element in the list has index 0, the second element has index 1, and so on. Suppose that you want to select the height of emma, the float 1.68. It's the fourth element, so it has index 3. To select it, you use 3 inside square brackets.
+After you've created your very own Python list, you'll need to know how to  information in the list. Python uses something called an index to do this. Have a look at the fam list again here. The first element in the list has index 0, the second element has index 1, and so on. Suppose that you want to select the height of emma, the float 1.68. It's the fourth element, so it has index 3. To select it, you use 3 inside square brackets.
```
Comment (adrian-datacamp) [inline scripts/chapter2_script.md]: "access" missing

File: scripts/chapter4_script.md [hunk 2]
```diff
@@ -50,13 +50,13 @@ If you ask for the type of these arrays, Python tells you that they are numpy.nd
 
 You can create a 2D numpy array from a regular Python list of lists. Let's try to create one numpy array for all height and weight data of your family, like this.
 
-If you print out np_2d now, you'll see that it is a rectangular data structure: Each sublist in the list, corresponds to a row in the two dimensional numpy array. From np_2d.shape, you can see that we indeed have 2 rows and 5 columns. shape is a so-called attribute of the np2d array, that can give you more information about what the data structure looks like.
+If you print out np_2d now, you'll see that it is a rectangular data structure: Each sublist in the list, corresponds to a row in the two dimensional numpy array. From np_2d.shape, you can see that we indeed have 2 rows and 5 columns. shape is a so-called attribute of the np2d array, that can give you more information about what the data structure looks like. [HBA] should we mention that attribute are NOT methods? many beginners trip up on this [/HBA]
```
Comment (adrian-datacamp) [inline scripts/chapter4_script.md]: As long as we keep it concise and don't go off topic, I think we should!

File: scripts/chapter4_script.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ Once you get used to it, this syntax is more intuitive and opens up more possibi
 
 Similarly, you can select the weight of all family members like this: you only want the second row, so put 1 before the comma. You want all columns, so you use a colon after the comma. The intersection gives us the entire second row.
 
-Finally, 2D numpy arrays enable you to do element-wise calculations, the same way you did it with 1D numpy arrays. That's something you can experiment with in the exercises, along with creating and subsetting 2D numpy arrays! Exciting...
+Finally, 2D numpy arrays enable you to do element-wise calculations, the same way you did it with 1D numpy arrays. That's something you can experiment with in the exercises, along with creating and subsetting 2D numpy arrays! Get coding, have fun and see you back here soon! While I wait, I may even code a bit myself.
```
Comment (adrian-datacamp) [inline scripts/chapter4_script.md]: 🤣 I love this

**Comment by adrian-datacamp** [comment]
Oh, and just to check as I haven't been involved in those conversations: @westonstearns, is changing the scripts to make them Hugo-style part of the plan for this A/B test?

**Comment by hugobowne** [comment]
@adrian-datacamp If you're able to edit the slides, I'd appreciate that. We haven't discussed process but my specs are to edit scripts & record. If you don't have time for the slides, we'll figure out somebody else to do this.

Also, note that if we mention conda, we may need to mention it earlier on. In a chapter 1 script we tell students how to download python but not using conda.

**Comment by hugobowne** [comment]
@adrian-datacamp i've made all edits discussed above.

later today, ill make some cosmetic changes to intros and outros, e.g.

"You're well on your way to becoming a Pythonista data science ninja!"   :D

**Comment by adrian-datacamp** [comment]
Awesome! Feel free to ping me to review the diff, if you'd like :)

**Comment by beccarobins** [comment]
@hugobowne Is this a PR we need to keep open for any reason? Can we close this and delete any related branches?

**Pull request closed without merging**